lib/crypto.mjs

```diff
@@ -0,0 +1,46 @@
+import crypto from 'crypto';
+
+export default crypto;
+
+export const createCipher = crypto.createCipher;
+export const createCipheriv = crypto.createCipheriv;
+export const createDecipher = crypto.createDecipher;
+export const createDecipheriv = crypto.createDecipheriv;
+export const createDiffieHellman = crypto.createDiffieHellman;
+export const createDiffieHellmanGroup = crypto.createDiffieHellmanGroup;
+export const createECDH = crypto.createECDH;
+export const createHash = crypto.createHash;
+export const createHmac = crypto.createHmac;
+export const createSign = crypto.createSign;
+export const createVerify = crypto.createVerify;
+export const getCiphers = crypto.getCiphers;
+export const getCurves = crypto.getCurves;
+export const getDiffieHellman = crypto.getDiffieHellman;
+export const getHashes = crypto.getHashes;
+export const pbkdf2 = crypto.pbkdf2;
+export const pbkdf2Sync = crypto.pbkdf2Sync;
+export const privateDecrypt = crypto.privateDecrypt;
+export const privateEncrypt = crypto.privateEncrypt;
+export const prng = crypto.prng;
```

Comment:
We should decide if we really want to export pure aliases or take the opportunity to begin limiting access to those

Reply:
can you expand on the reason to limit access to things?